docker-compose.yml

```diff
@@ -34,6 +34,7 @@ services:
           dbname: invidious
         full_refresh: false
         https_only: false
+        statistics_enabled: false
```

Comment:
This is not required, as the default value is `false` :)

Reply:
I had put it because I had tested by altering the parameter, but indeed there is no interest to leave it at its default value within the docker-compose